lib/ProteinStructureUtils/Utils/PDBUtils.py

```diff
@@ -74,15 +98,103 @@ def _file_to_data(self, file_path):
             'num_chains': chain_no,
             'num_residues': res_no,
             'num_atoms': atom_no,
-            'protein': {
+            'proteins': [{
```

Comment:
If it will always create exactly one `protein`, why do you want to put it into a list?

Reply:
The key is the `always`.  As of now that is what I have (one protein), but Chris said it could be more than one.